src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,26 @@
+$days-size: 100px;
+$gap: 1px;
+$padding: 10px;
+$transition: all 0.5s ease;
+$hundred-percent: 100%;
+$flex: flex;
+$center: center;
+$thirty-pixels: 30px;
+$none: none;
```

Comment:
it's a bit weird )))
-__- why? They're just the same.
don't over-introduce variables ))
```
$flex: flex;
$center: center;
$none: none;
```